src/panels/config/integrations/ha-integration-header.ts

```diff
@@ -122,10 +122,11 @@ export class HaIntegrationHeader extends LitElement {
     .header {
       display: flex;
       position: relative;
-      padding: 16px 8px 8px 16px;
+      padding: 0 8px 8px 16px;
     }
     .header img {
       margin-right: 16px;
+      padding-top: 16px;
```

Comment:
padding on an image feels weird. Why not margin-top?

Reply:
It already had it, so just moved it

Reply:
It had a padding on a `<div>`, not on `<img>`. We shouldn't add padding to an `<img>`. So let's make it margin, should work the same.